lib/bundler/fetcher.rb

```diff
@@ -99,7 +99,8 @@ def fetch_spec(spec)
 
       uri = URI.parse("#{remote_uri}#{Gem::MARSHAL_SPEC_DIR}#{spec_file_name}.rz")
       if uri.scheme == "file"
-        Bundler.load_marshal Bundler.rubygems.inflate(Gem.read_binary(uri.path))
+        path = Gem.win_platform? ? uri.path[1..-1] : uri.path
```

Comment:
So, I wanted to make this logic a bit more robust (not always assume windows parsed paths start with `/`), and I found that this logic is already implemented in rubygems:

https://github.com/rubygems/rubygems/blob/d470f1787a94f751c907e8299ec44f61b7689432/lib/rubygems/remote_fetcher.rb#L320-L327

So I guess I can reuse that logic here?

Reply:
Opened rubygems/rubygems#2767 to make this easier.

Reply:
I included using the linked PR here :+1: